frontend/src/components/utils/autoRefreshHandler.tsx

```diff
@@ -1,8 +1,14 @@
+import InfiniteScroll from "react-infinite-scroll-component";
+
 import React from "react";
 
+import PaginationSpinner from "./PaginationSpinner";
+
 export default (
   WrappedComponent: React.ComponentType<any>,
   fetchDataFn: Function,
+  count: number,
+  scroll: boolean,
```

Comment:
The API of this function leads to poor readability on the call site:

```
autoRefreshHandler(MyComponent, () => {}, 10, false, props);
```

Let's either expect a single argument (an object with all the parameters, like `props` to React components) or reduce the number of arguments to 3: component, config (an object which will contain fetchDataFn, "count" and "scroll"), props.